crds/workspace.devfile.io_devworkspaces.yaml

```diff
@@ -4605,6 +4612,118 @@ spec:
                           - componentClass
                           - embeddedResource
                           type: object
+                        image:
+                          description: Allows specifying the definition of an image
+                            for outer loop builds
+                          oneOf:
+                          - required:
+                            - dockerfile
+                          properties:
+                            dockerfile:
+                              description: Allows specifying dockerfile type build
+                              oneOf:
+                              - required:
+                                - uri
+                              - required:
+                                - registry
+                              - required:
+                                - git
+                              properties:
+                                args:
+                                  description: The arguments to supply to the dockerfile
+                                    build.
+                                  items:
+                                    type: string
+                                  type: array
+                                buildContext:
+                                  description: Path of source directory to establish
+                                    build context. Defaults to ${PROJECT_ROOT} in
+                                    the container
+                                  type: string
+                                git:
+                                  description: Dockerfile's Git source
+                                  properties:
+                                    checkoutFrom:
+                                      description: Defines from what the project should
+                                        be checked out. Required if there are more
+                                        than one remote configured
+                                      properties:
+                                        remote:
+                                          description: The remote name should be used
+                                            as init. Required if there are more than
+                                            one remote configured
+                                          type: string
+                                        revision:
+                                          description: The revision to checkout from.
+                                            Should be branch name, tag or commit id.
+                                            Default branch is used if missing or specified
+                                            revision is not found.
+                                          type: string
+                                      type: object
+                                    gitLocation:
```

Comment:
I would say it's really unclear field name...
From first look I thought it's Git Repo Location, but it's a relative location in git project file structure... Maybe analog to `podman build -f ` name like `file` or `fileLocation` could be better... We also may assume `Dockerfile` as a default value.

Reply:
I shall rename this to `fileLocation`

Reply:
updated!